test/kubemark/resources/start-kubemark-master.sh

```diff
@@ -123,7 +141,6 @@ function assemble-docker-flags {
 	# TODO(shyamjvs): Incorporate network plugin options, etc later.
 	echo "DOCKER_OPTS=\"${docker_opts}\"" > /etc/default/docker
 	echo "DOCKER_NOFILE=65536" >> /etc/default/docker  # For setting ulimit -n
-	service docker restart
```

Comment:
The gci configure-helper.sh wasn't restarting docker unless using a network plugin (which it was using anyway since "${NETWORK_PROVIDER:-}" == "kubenet"). Thus it restarts the docker daemon through systemctl always. However, in the case of kubemark I haven't made it restart following the gci's configure-helper.sh script. Have fixed this issue now in kubemark by restarting docker and I think that's a bug in configure-helper.sh that needs a fix (restarting even when not using network plugin).